core/src/main/java/org/mule/runtime/core/privileged/processor/MessageProcessors.java

```diff
@@ -167,10 +171,10 @@ public static CoreEvent processToApply(CoreEvent event, ReactiveProcessor proces
           .doOnError(completeErrorIfNeeded((event.getContext()), completeContext))
           .block();
     } catch (Throwable e) {
-      MuleException muleException = rxExceptionToMuleException(e);
       if (e.getCause() instanceof InterruptedException) {
         currentThread().interrupt();
       }
+      MuleException muleException = rxExceptionToMuleException(e);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Immediately throw this expression instead of assigning it to the temporary variable "muleException". ([squid:S1488](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1488))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7683&issues=AWjyMA29otIXnmsUSXGa&open=AWjyMA29otIXnmsUSXGa)

[ISSUE_KEY]: AWjyMA29otIXnmsUSXGa